ecl/hql/hqlattr.cpp

```diff
@@ -3989,8 +3988,8 @@ void CHqlRealExpression::addProperty(ExprPropKind kind, IInterface * value)
     SpinBlock block(*propertyLock);
```

Comment:
Did you mean to leave this line here?

Reply:
I agree

Reply:
Yes - they are protecting adding an item to the list, but once they are added they are never removed.  I could try and be clever using compare_exchange to add the items, but I didn't think it was worth it.